src/ipc/Coordinator.cc

```diff
@@ -143,6 +148,21 @@ void Ipc::Coordinator::handleRegistrationRequest(const HereIamMessage& msg)
     SendMessage(MakeAddr(strandAddrLabel, msg.strand.kidId), message);
 }
 
+void
+Ipc::Coordinator::handleForegroundRebuildMessage(const StrandMessage& msg)
+{
+    // notify any searchers waiting for this strand
+    for (const auto searchRequest: searchers) {
+        if (searchRequest.tag != msg.strand.tag)
+            continue;
+
+        Ipc::TypedMsgHdr message;
+        message.setType(Ipc::mtStrandBusy);
```

Comment:
If possible, let's avoid foo inside foo:

```suggestion
        TypedMsgHdr message;
        message.setType(mtStrandBusy);
```

As always, please check for similar problems in the new/changed PR code.